2017/ru/0xd1-data-contributors.md

```diff
@@ -0,0 +1,136 @@
+# **Благодарности**
+
+## Организации, предоставившие данные
+
+Хотим поблагодарить организации, которые предоставили свои данные по уязвимостям для выпуска обновленной версии 2017 года:
+
+* ANCAP
+* Aspect Security
+* AsTech Consulting
+* Atos
+* Branding Brand
+* Bugcrowd
+* BUGemot
+* CDAC
+* Checkmarx
+* Colegio LaSalle Monteria
+* Company.com
+* ContextIS
+* Contrast Security
+* DDoS.com
+* Derek Weeks
+* Easybss
+* Edgescan
+* EVRY
+* EZI
+* Hamed
+* Hidden
+* I4 Consulting
+* iBLISS Seguran̤a & Intelig̻encia
+* ITsec Security Services bv
+* Khallagh
+* Linden Lab
+* M. Limacher IT Dienstleistungen
+* Micro Focus Fortify
+* Minded Security
+* National Center for Cyber Security Technology
+* Network Test Labs Inc.
+* Osampa
+* Paladion Networks
+* Purpletalk
+* Secure Network
+* Shape Security
+* SHCP
+* Softtek
+* Synopsis
+* TCS
+* Vantage Point
+* Veracode
+* Web.com
+
+Впервые все данные, предоставленные для выпуска Топ-10, а также полный список источников [доступны публично](https://github.com/OWASP/Top10/tree/master/2017/datacall/submissions).
+
+## Отдельные участники проекта
+
+Хотим поблагодарить отдельных участников проекта, которые внесли ощутимый вклад в работу по созданию Топ-10 на GitHub:
+
+* ak47gen
+* alonergan
+* ameft
+* anantshri
+* bandrzej
+* bchurchill
+* binarious
+* bkimminich
+* Boberski
+* borischen
+* Calico90
+* chrish
+* clerkendweller
+* D00gs
+* davewichers
+* drkknight
+* drwetter
+* dune73
+* ecbftw
+* einsweniger
+* ekobrin
+* eoftedal
+* frohoff
+* fzipi
+* gebl
+* Gilc83
+* gilzow
+* global4g
+* grnd
+* h3xstream
+* hiralph
+* HoLyVieR
+* ilatypov
+* irbishop
+* itscooper
+* ivanr
+* jeremylong
+* jhaddix
+* jmanico
+* joaomatosf
+* jrmithdobbs
+* jsteven
+* jvehent
+* katyanton
+* kerberosmansour
+* koto
+* m8urnett
+* mwcoates
+* neo00
+* nickthetait
+* ninedter
+* ossie-git
+* PauloASilva
+* PeterMosmans
+* pontocom
+* psiinon
+* pwntester
+* raesene
+* riramar
+* ruroot
+* securestep9
+* securitybits
+* SPoint42
+* sreenathsasikumar
+* starbuck3000
+* stefanb
+* sumitagarwalusa
+* taprootsec
+* tghosth
+* TheJambo
+* thesp0nge
+* toddgrotenhuis
+* troymarshall
+* tsohlacol
+* vdbaan
+* yohgaki
```

Comment:
You've forgotten to add yourself to this list (@JZDLin, @oxdef), haven't you? 😄

Reply:
No, no, no. This is the list of vulnerability-related data contributors.